library/Icinga/Application/Config.php

```diff
@@ -412,17 +412,16 @@ public static function app($configname = 'config', $fromDisk = false)
      */
     public static function module($modulename, $configname = 'config', $fromDisk = false)
     {
-        if (! isset(self::$modules[$modulename])) {
-            self::$modules[$modulename] = array();
+        if (! isset(static::$modules[$modulename])) {
+            static::$modules[$modulename] = array();
```

Comment:
Good point Tom :) I just thought use static so you don't have to wrap around your head whether to use self or static. But I like the rule of thumb to use self for props and consts and static for method calls and factory methods.